packages/website/docs/docs/theming.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Theming & Styling
+description: BlockNote allows you to change how the editor UI looks. You can change the theme of the default UI, or override its CSS styles.
+imageTitle: Theming & Styling
+path: /docs/theming
+---
+
+<script setup>
+import { useData } from 'vitepress';
+import { getTheme, getStyles } from "./demoUtils";
+
+const { isDark } = useData();
+</script>
+
+# Theming & Styling
+
+BlockNote allows you to change how the editor UI looks. You can change the theme of the default UI, or override its CSS styles.
+
+## Theming
+
+If you want to make small adjustments to BlockNote's UI, you can create a custom theme, which will let you change colors, fonts, and border radii. You can do this by passing partial `Theme` objects to `BlockNoteView`.
+
+Take a look at how this is done in the demo below:
+
+::: sandbox {template=react-ts}
```

Comment:
For sake of simplicity, I would just make one "red theme" and not a red light / dark theme

Reply:
I think it's nice to have both since it both shows off the auto switching from browser settings and saves us having to explain how to pass in a light & dark theme instead of just a single theme. Theming is also not really complicated compared to a lot of the other stuff, since 90% of it is defining colors, so imo we shouldn't run into the issue of people getting overloaded with information.